### PR TITLE
OS WGLC Review

### PR DESCRIPTION
**Pull request by OR13**
WGLC Review

### CODE REVIEW

File: draft-ietf-scitt-software-use-cases.md
Comment (aj-stein-nist) [inline draft-ietf-scitt-software-use-cases.md]: Minor, non-blocking: spelling.

```suggestion
Supply chain security has historically focused on risk management practices to safeguard logistics, meet compliance regulations, forecast demand, and optimize inventory.
```

File: draft-ietf-scitt-software-use-cases.md
Comment (aj-stein-nist) [inline draft-ietf-scitt-software-use-cases.md:311]: Minor, non-blocking: I am cool with this but generalized it because I felt even consumer laptops fall in this category with UEFI initialization as a common use case that is like IoT but not IoT.

File: draft-ietf-scitt-software-use-cases.md
Comment (aj-stein-nist) [inline draft-ietf-scitt-software-use-cases.md]: Minor, non-blocking: I avoided getting into this before during my previous review and recommended edits, but compliance with what? With law? With finance regulations? With cybersecurity frameworks? The madness of this word is it understood to be important but without pointing out the what and why. I was tempted to remove it, but maybe we can be better about what and why?

```suggestion
SCITT-enabled products assist in managing compliance with often distinct, but overlapping and interconnected, legal, regulatory, and technical requirements, assessing risks, and detecting supply chain attacks across the software lifecycle while prioritizing data privacy.
```

**Review by aj-stein-nist — APPROVED** [review]
I really like how you tightened up the wording and got it punchy with more active language. I found one typo and have two minor quibbles, but I can let go of those.

File: draft-ietf-scitt-software-use-cases.md
Comment (OR13) [reply draft-ietf-scitt-software-use-cases.md:311]: yeah, as I said on the list, I feel these use cases need more than just phrasing updates... I am not going to attempt that in this PR.

File: draft-ietf-scitt-software-use-cases.md
Comment (aj-stein-nist) [reply draft-ietf-scitt-software-use-cases.md]: Re you previous comment, we are good here. Resolve this. We can revise later outside of the WGLC review process as needed.

File: draft-ietf-scitt-software-use-cases.md
Comment (aj-stein-nist) [reply draft-ietf-scitt-software-use-cases.md:311]: Agreed, you can disregard this one (and the other, since it seems I cannot resolve the comment).

File: draft-ietf-scitt-software-use-cases.md
Comment (SteveLasker) [inline draft-ietf-scitt-software-use-cases.md]: nit:
```suggestion
Firmware is often the bedrock on which the security story of a device is built.
```

File: draft-ietf-scitt-software-use-cases.md
Comment (SteveLasker) [inline draft-ietf-scitt-software-use-cases.md]: what is the `fixme` reference?

**Review by SteveLasker — DISMISSED** [review]
Approve with some minor nits

**Comment by rjb4standards** [comment]
No objection to these proposed changes. Well done, Orie.

**Review by rjb4standards — APPROVED** [review]
No objection to the proposed changes,

Dick Brooks (REA)

File: draft-ietf-scitt-software-use-cases.md
Comment (henkbirkholz) [reply draft-ietf-scitt-software-use-cases.md]: attempted a fix in commit 5865af8759e5a8fc45e1afb0b57247efea03b483

**Review by henkbirkholz — DISMISSED** [review]
LGTM!

**Comment by bobmartin3000** [comment]
Changes look good.

**Review by bobmartin3000 — APPROVED** [review]
Ready to go.

**Review by SteveLasker — DISMISSED** [review]
lgtm

File: draft-ietf-scitt-software-use-cases.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
These use cases aim to identify software supply chain problems that the industry faces today and act as a guideline for developing a comprehensive security architecture  and solutions for these scenarios.
```

File: draft-ietf-scitt-software-use-cases.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
- store RIMs in a secure fashion that enables their usage in appraisal procedures years after they were created
```

File: draft-ietf-scitt-software-use-cases.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
Relevant documents (such as compliance, requirements or procedure results) originate from various sources and include a wide range of representations and formats.
```

File: draft-ietf-scitt-software-use-cases.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
If the off-line systems in operation are part of constrained node environments, they do not possess the capabilities to process and evaluate all the authenticity proofs that come with the released software.
```

**Comment by yogeshbdeshpande** [comment]
@OR13 : As you are on a `forked` repo, I cannot make the changes which I have mentioned here : Directly.

Could you please `merge` my suggested .Nits so that I can merge the PR.

Thanks again for all the hardwork of reviewing the document!

File: draft-ietf-scitt-software-use-cases.md
Comment (bobmartin3000) [reply draft-ietf-scitt-software-use-cases.md]: agree

File: draft-ietf-scitt-software-use-cases.md
Comment (bobmartin3000) [reply draft-ietf-scitt-software-use-cases.md]: not sure what "believable" means here, agree on secure.

File: draft-ietf-scitt-software-use-cases.md
Comment (bobmartin3000) [reply draft-ietf-scitt-software-use-cases.md]: agree

File: draft-ietf-scitt-software-use-cases.md
Comment (bobmartin3000) [reply draft-ietf-scitt-software-use-cases.md]: agree

**Comment by OR13** [comment]
@yogeshbdeshpande done, thanks for your suggestions, I have applied them all.

This PR still requires at least 1 approval from code owners to be merged.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Comment by SteveLasker** [comment]
@yogeshbdeshpande, as an author of the doc, can you LGTM to affirm you agree with the merged changes?

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Comment by rjb4standards** [comment]
I'll bite, what's an LGTM, Steve and is it available on Amazon Prime?

**Comment by yogeshbdeshpande** [comment]
@SteveLasker : I will fix the failing lints

**Comment by yogeshbdeshpande** [comment]
> I'll bite, what's an LGTM, Steve and is it available on Amazon Prime?

LGTM=> Looks Good to Me :)

**Pull request merged**
